packages/components/.stylelintrc.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "@synergy-design-system/stylelint-config-syn"
+  "extends": "@synergy-design-system/stylelint-config-syn",
+  "rules": {
```

Comment:
As this will also happen when developing components outside of @synergy scope and we would like to publish `@synergy-design-system/stylelint-config-syn`, I think it would be nice if you could add this to the ruleset of the stylelint-package in `_private` instead of the package directly. With that, we are more future proof.